src/main/scala/snabbdom/hFunction.scala

```diff
@@ -57,13 +61,14 @@ trait DataObject extends js.Object {
   val style: js.Dictionary[String]
   val on: js.Dictionary[js.Function1[Event, Unit]]
   val hook: Hooks
-  val key: js.UndefOr[String | Int]
+  val key: js.UndefOr[KeyValue]
 }
 
 object DataObject {
 
   type PropValue = Any
   type AttrValue = String | Boolean
+  type KeyValue = String | Int
```

Comment:
Where is this definition coming from?

Reply:
https://github.com/snabbdom/snabbdom#key--string--number

Reply:
And should we use `Double` instead of `Int` here? (because of `Number`) Javascript only knows `Double` anyways, so this should be a noop even when using `Int`-keys.